manual/src/cmds/intf-c.etex

```diff
@@ -727,8 +727,8 @@ $\hbox{"string_length"}(v)-1$.
 or byte sequence \var{v}, with type "unsigned char". Bytes are
 numbered from 0 to $\hbox{"string_length"}(v)-1$.
 \item "String_val("\var{v}")" returns a pointer to the first byte of the string
-\var{v}, with type "char *" or, when OCaml is configured with
-"-force-safe-string", with type "const char *".
+\var{v}, with type "const char *" since strings can only be immutable from
+OCaml 5.0 onwards.
```

Comment:
I would remove the historical mention, which I think is slightly confusing and not so important here.